Testing/WinMLRunnerTest/WinMLRunnerTest.cpp

```diff
@@ -747,45 +801,45 @@ namespace WinMLRunnerTest
             const std::wstring command = BuildCommand({ EXE_PATH, L"-model", modelPath, L"-input", inputPath });
             Assert::AreEqual(HRESULT_FROM_WIN32(ERROR_INVALID_PARAMETER), RunProc((wchar_t *)command.c_str()));
         }
-        TEST_METHOD(ProvidedCSVInputGPUSaveTensor)
-        {
+        TEST_METHOD_WITH_NAME(ProvidedCSVInputSaveTensor)
```

Comment:
The test name keep GPU in its name so that it is easy to tell that the test runs WinMLRunner using GPU as device.

Reply:
May have been a merge error.  Fixed.